lib/rubocop/cop/lint/assignment_in_condition.rb

```diff
@@ -41,6 +48,14 @@ def on_until(node)
 
         private
 
+        def message(node)
```

Comment:
There's a RuboCop offense here, failing the build. Since `node` is not used, it should be `_node`. 🙂